site/docs/get-started/super-quickstart.md

```diff
@@ -0,0 +1,45 @@
+# Super Quick Start Guide
+
+this guide contains a script that you can run with a single command to see a demo of Fybrik in action.  
+
+this demo demonstrates the following sequence -  
+
+1. A **data operator** boots up Fybrik and registers a data asset of financial data to Fybrik
+
+2. A **data governance officer** defines data policies, such as which columns contain PII (personally identifiable information), and submits them to Fybrik
+
+3. A **data user** submits a request to read the data asset using Fybrik
+
+4. **Fybrik** fetches the data asset, and automatically redacts columns according to the data policies
+
+5. The **data user** can consume the governed data instantly
+
+All your require for this demo is a working bash terminal.  
+The demo will make a bin folder at your current directory with all the required dependencies for Fybrik.
+
+## Demo with OpenMetaData data catlog (~25 mintues)
+For a demo with the current version of Fybrik using its main data catlog OpenMetaData, you can run the following command.  
+Note: booting up Fybrik with openmetadata takes ~25 minutes on most machines, so grab a coffee while you wait!  
+Alternatively you can run the [next](#demo-with-katalog-a-data-catalog-stub-5-mintues) script which boots up Fybrik with a data catalog stub.  
+
+```bash
+curl https://raw.githubusercontent.com/aradhalevy/fybrik/SuperQucikStart/samples/super-quickstart/SuperQuickStart-OMD.sh | bash -
+```
+
+## Demo with Katalog, a data catalog stub (~5 mintues)
+For a demo using an older version of Fybrik, without its main Data Catlog, you can run the following command.  
+this demo runs using Katalog, a data catalog stub, strictly used for demos, testing and evaluation purposes.
+
+```bash
+curl https://raw.githubusercontent.com/aradhalevy/fybrik/SuperQucikStart/samples/super-quickstart/SuperQuickStart-Katalog.sh | bash -
+```
+
+## Cleanup
+
+To stop local kubernetes cluster booted up on your machine for this demo, and to remove the folder created with the dependencies for Fybrik, run this.  
+warning - if you already had a bin folder at your current directory these commands will delete it and its contents
+
+```bash
+bin/kind delete clusters kind-fybrik-installation-sample
```

Comment:
why do you use `delete clusters` instead of 'delete cluster` ?
this cleanup will work only in the case of kind deployment. We need to add comments about it

Reply:
its a KIND syntax thing, if you use delete cluster you need to add a --name flag. 
changed to `bin/kind delete cluster --name=kind-fybrik-installation-sample
`
for readbility